src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -512,6 +512,10 @@ private void parseApplication(Document doc, XPathExpression expression) throws X
             // add unique values only
             if (!nodeValue.isEmpty()) {
                 if(expression.equals(XPATH_SERVER_SPRINGBOOT_APPLICATION)){
+                    // checking whether any springBootAppNodeLocation already configured from other server configuration files
+                    if(springBootAppNodeLocation.isPresent()){
+                        throw new PluginExecutionException("Found multiple springBootApplication elements specified in the server configuration. Only one springBootApplication can be configured per Liberty server.");
+                    }
```

Comment:
I think we should include the path of the config file where the springBootApplication element was found in the error message. Also include in above message at line 508, and add an `else` to this `if` to print a log.debug statement with the config file location of the first springBootApplication element found.